### PR TITLE
Fix mobile users experience with tilemap painting

### PR DESCRIPTION
**Pull request by AlexandreSi**
Don't show in changelog

### CODE REVIEW

File: newIDE/app/src/InstancesEditor/TileSetVisualizer.js [hunk 1]
```diff
@@ -554,8 +548,87 @@ const TileSetVisualizer = ({
     [displayedTileSize, columnCount, rowCount]
   );
 
+  // const getTooltipRelativePosition = React.useCallback(
```
Comment (4ian) [inline newIDE/app/src/InstancesEditor/TileSetVisualizer.js:551]: To remove?

File: newIDE/app/src/InstancesEditor/TileSetVisualizer.js [hunk 2]
```diff
@@ -664,101 +737,96 @@ const TileSetVisualizer = ({
...
-                  columnCount,
-                }).toString()}
+            <div
+              style={styles.tilesetContainer}
+              ref={tilesetContainerRef}
+              {...(interactive ? interactionCallbacks : undefined)}
+              {...longTouchProps}
+              onMouseMove={onMouseMove}
+              onMouseEnter={onMouseEnter}
+              onMouseLeave={onMouseLeave}
```
Comment (4ian) [inline newIDE/app/src/InstancesEditor/TileSetVisualizer.js:750]: Could/should this be onPointerXxx?

Comment (AlexandreSi) [reply newIDE/app/src/InstancesEditor/TileSetVisualizer.js:750]: To handle the tooltip display, either you're on desktop and it watches the mouse, either on mobile with the long touch props.
Before that, it was using the `title` attribute of the tile components but it's too slow to appear so I handled it with JS

File: newIDE/app/src/InstancesEditor/TileSetVisualizer.js [hunk 3]
```diff
@@ -166,14 +172,10 @@ const Tile = ({
   width = 1,
   height = 1,
   highlighted,
-  title,
-  displayTooltip,
 }: TileProps) => {
   const classes = useStylesForTile(!!highlighted);
   // Tooltip position has to be adapted because the image is in a overflow auto parent
```
[the remark this answers is not in the document]
Comment (AlexandreSi) [reply newIDE/app/src/InstancesEditor/TileSetVisualizer.js:177]: Nope, good catch!

File: newIDE/app/src/InstancesEditor/TileSetVisualizer.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/InstancesEditor/TileSetVisualizer.js:730]: I'm usually not a big fan of making custom tooltips, but let's go ahead with this.

**Review by 4ian — APPROVED** [review]

**Pull request merged**